libraries/botframework-schema/src/index.ts

```diff
@@ -830,6 +830,16 @@ export interface ActivityTimestamps extends Activity {
     rawLocalTimestamp?: string;
 }
 
+export const conversationParametersObject = z.object({
+    isGroup: z.boolean(),
+    bot: channelAccount,
+    members: z.array(channelAccount).optional(),
+    topicName: z.string().optional(),
+    tenantId: z.string().optional(),
+    activity: activity,
+    channelData: z.unknown().optional(),
+});
+
```

Comment:
This could be removed if the previously mentioned case is changed.

Reply:
it can't, TS requires an exact parse to extract the property required to save the parameters in the property desired